core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -520,7 +520,7 @@ private String transformOMAScore( String omaScore) {
         } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
             return "N";
         } else if( omaScore.equalsIgnoreCase("[sent]")) {
-            return ExtendedMutationUtil.NOT_AVAILABLE; // TODO temp workaround for invalid sent values
+            return "NA"; // TODO temp workaround for invalid sent values
```

Comment:
This method was using the `ExtendedMutationUtil.NOT_AVALABLE` constant without being related to the `HGVSp_Short`-empty values. 

I changed the `HGVSp_Short`-empty values from a string to a list, so this method had some trouble what to return then. The other lines in this method return a character directly, so I changed this line to do that as well. See:

```
    private String transformOMAScore( String omaScore) {
        if( omaScore == null || omaScore.length() ==0) {
            return omaScore;
        }
        if( omaScore.equalsIgnoreCase("H") || omaScore.equalsIgnoreCase("high")) {
            return "H";
        } else if( omaScore.equalsIgnoreCase("M") || omaScore.equalsIgnoreCase("medium")) {
            return "M";
        } else if( omaScore.equalsIgnoreCase("L") || omaScore.equalsIgnoreCase("low")) {
            return "L";
        } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
            return "N";
        } else if( omaScore.equalsIgnoreCase("[sent]")) {
            return "NA"; // TODO temp workaround for invalid sent values
        } else {
            return omaScore;
        }
    }
```